src/preprocessing/passes/normalize.h

```diff
@@ -0,0 +1,97 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Daneshvar Amrollahi
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2023 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * 
+ */
+
+ #include "cvc5_private.h"
+
+ #ifndef CVC5__PREPROCESSING__PASSES__NORMALIZE_H
+ #define CVC5__PREPROCESSING__PASSES__NORMALIZE_H
+ 
+ #include "preprocessing/preprocessing_pass.h"
+ #include <unordered_map>
+ 
+ namespace cvc5::internal {
+ namespace preprocessing {
+ namespace passes {
+ 
+ 
+ 
+ struct NodeInfo
```

Comment:
Give comments on all classes, data members and methods. See style guide, we use comments enclosed in `/** ... */` in all places.